src/modules/poweraccent/PowerAccent.Core/Services/SourceGenerationContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json.Serialization;
+
+namespace PowerAccent.Core.Services
+{
+    [JsonSourceGenerationOptions(WriteIndented = true)]
+    [JsonSerializable(typeof(SettingsService))]
+    public partial class SourceGenerationContext : JsonSerializerContext
```

Comment:
I believe we should use a common `Helpers` or `Contexts` folder to put these `JsonSerializerContext` classes underneath, as there may ones that encompass multiple `JsonSerializable` types.

Reply:
@moooyo  - Other than this, change, it LGTM.

Reply:
Agree. any suggestion? How about move to ManagedCommon? @snickler

Reply:
Just one confuse, It seems that we cannot centrailize it? Let me explain why I think that.

Consider we have a sub-project named JsonContextCommon, which include all the jsonSerializerContext we need.
Then, if we add a new jsonSerializerContext, we also need to add the [JsonSerializeable(typeof(T))]. Which require us to ref the Type T sub-project in the JsonContextCommon sub-project.
Now, we need to use this context. So, we need to ref JsonContextCommon sub-project in the project which we want to use it. We will necounter cycle reference error.

Only if we move all Type T to a independent sub-project such as ModelCommon, can we move the JsonSerializerContext to the same centralized location.

@snickler

Reply:
@mooyo - In terms of centralization, I meant per project. The JsonSerializerContext classes created should have common folder placement and structure for consistency.

Reply:
I agree. Thanks @snickler